apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts

```diff
@@ -1505,4 +1513,21 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
             this.adjustedReceiptsAmountChange();
         }
     }
+    /**
+     * Advance receipt adjustment remove model action response
+     *
+     * @param {*} userResponse  Action message
+     * @memberof UpdateLedgerEntryPanelComponent
+     */
+    public onAdvanceReceiptRemoveCloseConfirmationModal(userResponse: any): void {
+        if (userResponse.response) {
```

Comment:
This method can be made short as:
`
if (userResponse) {
this.isAdvanceReceipt = !userResponse.response;
this.handleAdvanceReceiptChange();
this.advanceReceiptRemoveConfirmationModal.hide();
}
`